modules/iam-users/main.tf

```diff
@@ -182,7 +182,7 @@ resource "aws_lambda_permission" "allow_cloudwatch_every_five_minutes" {
 }
 
 # Permission boundary
-resource "aws_iam_policy" "s3_full_access_boundary" {
+resource "aws_iam_policy" "user_access_boundary" {
```

Comment:
I think this may cause issues. It would attempt to destroy and recreate the policy which is in use by resources not management by terraform and fail. L

Let's leave this as is for now.

Reply:
Oh okay, sounds good